pandas/tests/frame/test_constructors.py

```diff
@@ -2679,3 +2679,7 @@ def test_construction_from_set_raises(self):
         msg = "Set type is unordered"
         with pytest.raises(TypeError, match=msg):
             pd.DataFrame({"a": {1, 2, 3}})
+
+    def test_construct_empty_dataframe_with_string_dtype(self):
```

Comment:
move this to pandas/tests/extension/test_common.py and use the fixtures

Reply:
Thanks @jreback. I move this test to That file.